docs/Too-Many-Methods.md

```diff
@@ -45,7 +45,7 @@ class Smelly
 end
 ```
 
-would cause Reek to emit the same warning as in the example above.
+would not cause Reek to emit the warning in the example above.
```

Comment:
This now doesnt make sense anymore and is very confusing to new users. You're basically saying what Reek now is NOT doing when documentation should tell you what it IS doing ;)
It would be great if you could invert the whole paragraph here.